src/Accordion/Accordion.tsx

```diff
@@ -30,40 +25,45 @@ export interface AccordionProps {
     alternative?: boolean;
 }
 
+interface TransitionProps {
+    height?: string;
+    transition?: string;
+    dataCollapsed?: boolean;
+    ariaExpanded?: boolean;
+}
+
 const Accordion: React.FunctionComponent<AccordionProps> = (props: AccordionProps) => {
+    const collapsableRef: React.MutableRefObject<Array<React.RefObject<HTMLDivElement>>> = React.useRef(props.list.map(() => React.createRef<HTMLDivElement>()));
+    const [transitionProperties, setTransitionProperty] = React.useState<Array<TransitionProps>>(() => props.list.map((): TransitionProps => ({
```

Comment:
it's better to make `transitionProperties` state variable consistent. In this case, `setTransitionProperty` should be `setTransitionProperties`

Reply:
@kherP setTransitionProperty  update the transition property of a single item in the array of items , hence the singularity in the wording.